### PR TITLE
Layout and basic element styles for the ballot pages (all the data pages)

### PR DESCRIPTION
**Pull request by adborden**
Get's us a lot closer with the major layout components for the ballot pages.

On some of the pages, the ballot and locality aren't assigned, so the the ballot nav and page header show up empty.

Ballot/election page with ballot navigation

Mobile version of the ballot page

Office elections in the ballot nav

Referendum page with empty ballot nav

Mobile version of the candidate page

### CODE REVIEW

**Comment by adborden** [comment]
I should add that I'm not all that confident in the grids, specifically the `l-ballot` layout, how the grid is put together and how the sidebar fits into the grid. Happy to chat more, but if there's a CSS expert, I'd love to have them take a closer look here.

File: _includes/referendum_measure_heading.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{% case include.referendum_measure_display -%}
+{% when 'referendum' %}
+Referendums
+{% when 'measure' %}
+Ballot measures
+{% when 'proposition' %}
+Propositions
```
Comment (tdooner) [inline _includes/referendum_measure_heading.html]: Not sure I understand the difference between referendum, measure, and proposition.

File: _layouts/locality.html [hunk 2]
```diff
@@ -1,60 +1,14 @@
...
-      {% for ballot in ballots %}
-      <li><a href="{{ ballot.url | prepend: site.baseurl }}">{{ ballot.election | date: "%B %-d, %Y" }}</a></li>
-      {% endfor %}
-    </ul>
-    </div>
-  </div>
-</article>
+{% capture body %}
+<span>{{ locality.name }}-wide finance data coming soon! Choose a ballot item from the navigation.</span>
+{% endcapture %}
```
Comment (tdooner) [inline _layouts/locality.html]: Woah, what. Interesting.

File: _localities/san-francisco.md [hunk 3]
```diff
@@ -4,4 +4,5 @@ netfile: SFO
 name: San Francisco
 full_name: City and County of San Francisco
 current_election: 2018-06-05
+referendum_measure_display: referendum
```
Comment (tdooner) [inline _localities/san-francisco.md]: Oh I see.

**Pull request merged**